moon/powerups/server/grenadier.moon

```diff
@@ -17,9 +17,7 @@ class GrenadierPowerup extends BasePowerup
     new: (ply) =>
         super ply
 
-        duration = getConf "grenadier_duration"
-        @durationTimer = "CFC_Powerups-Grenadier-#{ply\SteamID64!}"
-        timer.Create @durationTimer, duration, 1, -> @Remove!
+        @usesRemaining = getConf "grenadier_uses"
```

Comment:
You have mixed casing here. Is it `usesRemaining` or `UsesRemaining`

Reply:
So does `magnetic_crossbow.moon`, the mixed casing came from ctrl-c'ing your code, so I'll fix it for both